subsys/nrf_security/src/drivers/cracen/Kconfig

```diff
@@ -31,18 +31,26 @@ config CRACEN_LIB_KMU
 	help
 	  The CRACEN KMU library.
 
-config CRACEN_LOAD_KMU_SEED
-	bool
-	prompt "Load the CRACEN IKG seed"
+config CRACEN_IKG_SEED_LOAD
+	bool "Load the CRACEN IKG seed"
 	depends on CRACEN_LIB_KMU
 	default y
 	help
-	  Configuration to load a The CRACEN IKG seed value from KMU.
-	  The KMU seed is used to give support for key derived from IKG.
+	  The IKG seed is used to derive the keys in the IKG.
 	  If there is no seed in KMU, it is expected to be generated.
 	  Note that this is not required to be used if a bootloader is loading
 	  KMU seed.
 
+config CRACEN_IKG_SEED_KMU_SLOT
+	int "KMU slot of the CRACEN IKG seed"
+	range 0 183
+	default 183
+	help
+	  The Cracen IKG seed spans over 3 KMU slots:
+	  the one defined here plus the two following it.
```

Comment:
Would it be better to say "and the two subsequent ones" instead?

Reply:
Fine to me.